packages/vue/src/helpers.ts

```diff
@@ -109,3 +109,13 @@ export const simpleMappers = (subState: string, props: any, getter: any) => func
 
   return slices.reduce((result, prop) => Object.assign({}, result, { [prop]: getter(prop)(state) }), {});
 };
+
+
+/**
+ * vue-router base name was changed in v4 from `base` to `history.base`
+ * @param router VueRouter v3 or v4
+ * @returns router base name
+ */
+export const getRouterBaseName = (router?: VueRouter): string | undefined => {
+  return router?.options.base || (router?.options as any).history?.base
```

Comment:
```suggestion
  return router?.options.base ?? (router?.options as any).history?.base
```

Reply:
I just extracted the logic, I don't wanna change it